xio-test/src/main/java/com/xjeffrose/xio/test/OkHttpUnsafe.java

```diff
@@ -0,0 +1,112 @@
+package com.xjeffrose.xio.test;
+
+import java.security.KeyManagementException;
+import java.security.KeyStore;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSession;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import okhttp3.OkHttpClient;
+import okhttp3.Protocol;
+import okhttp3.mockwebserver.MockWebServer;
+
+public class OkHttpUnsafe {
```

Comment:
I feel like this and the public accessors need to have very scary docs telling people never to use them outside of testing. . .